evm/src/executor/mod.rs

```diff
@@ -594,7 +596,7 @@ pub enum EvmError {
         traces: Option<CallTraceArena>,
         debug: Option<DebugArena>,
         labels: BTreeMap<Address, String>,
-        transactions: Option<VecDeque<TypedTransaction>>,
+        transactions: Option<VecDeque<BroadcastableTransaction>>,
```

Comment:
`VecDeque<BroadcastableTransaction>` is used a lot,

does it make sense to make a new type for this as well?